tests/framework/pvc.go

```diff
@@ -0,0 +1,47 @@
+package framework
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+func (f *Framework) CreatePVCFromDefinition(def *k8sv1.PersistentVolumeClaim) (*k8sv1.PersistentVolumeClaim, error) {
+	return utils.CreatePVCFromDefinition(f.K8sClient, f.Namespace.Name, def)
+}
+
+func (f *Framework) DeletePVC(pvc *k8sv1.PersistentVolumeClaim) error {
+	return utils.DeletePVC(f.K8sClient, f.Namespace.Name, pvc)
+}
+
+func (f *Framework) WaitForPersistentVolumeClaimPhase(phase k8sv1.PersistentVolumeClaimPhase, pvcName string) error {
+	return utils.WaitForPersistentVolumeClaimPhase(f.K8sClient, f.Namespace.Name, phase, pvcName)
+}
+
+func (f *Framework) CreateExecutorPodWithPVC(podName string, pvc *k8sv1.PersistentVolumeClaim) (*k8sv1.Pod, error) {
```

Comment:
This function crossed over into Pods and PVCs, so not sure if I should move that into a separate Pod file or keep it here. I kept it here for now, but I am not opposed to moving it into a pod.go with the other related Pod functions.

Reply:
If no clear answer then leave it where it is now.

Reply:
:+1: